src/main/java/mods/eln/Eln.java

```diff
@@ -5881,32 +6169,33 @@ private void recipeBattery() {
             'I', new ItemStack(Items.iron_ingot));
 
         addRecipe(findItemStack("Capacity Oriented Battery"),
-            "PPP",
             "PBP",
-            "PPP",
             'B', findItemStack("Cost Oriented Battery"),
             'P', "ingotLead");
 
         addRecipe(findItemStack("Voltage Oriented Battery"),
-            "PPP",
             "PBP",
-            "PPP",
             'B', findItemStack("Cost Oriented Battery"),
-            'P', new ItemStack(Items.iron_ingot));
+            'P', findItemStack("Iron Cable")); 
 
         addRecipe(findItemStack("Current Oriented Battery"),
-            "PPP",
             "PBP",
-            "PPP",
             'B', findItemStack("Cost Oriented Battery"),
             'P', "ingotCopper");
 
         addRecipe(findItemStack("Life Oriented Battery"),
-            "P P",
-            " B ",
-            "P P",
+            "PBP",
             'B', findItemStack("Cost Oriented Battery"),
             'P', new ItemStack(Items.gold_ingot));
+        addRecipe(findItemStack("Experimental Battery"),
```

Comment:
Heh. "I want you to take *all* of the batteries and put them in one" "What?" "Yeah! It will improve usefulness 10 fold!"

+1

Reply:
It's not quite as useful as it might seem though! You'll find out why once you try it out.